test/app.test.js

```diff
@@ -249,6 +249,70 @@ test.cb('should handle req/res request', t => {
   })
 })
 
+test.cb('should handle multiple protos request', t => {
+  t.plan(5)
+  const APP_HOST = tu.getHost()
+  const PROTO_ROOT_FOLDER = path.resolve(__dirname, './protos')
+  const PROTO_ROOT_MULTIPLE = path.resolve(__dirname, './protosnewroute')
+  const PROTO_PATH = path.resolve(__dirname, './protos/helloworld.proto')
+
+  function sayHello (ctx) {
+    ctx.res = { message: 'Hello ' + ctx.req.name }
+  }
+
+  const app = new Mali()
+  t.truthy(app)
+  console.log(PROTO_ROOT_MULTIPLE)
+  console.log(PROTO_ROOT_FOLDER)
```

Comment:
Please remove the unnecessary console.logs

Reply:
Removed!